packages/devextreme/testing/tests/DevExpress.ui.widgets/multiView.tests.js

```diff
@@ -1092,6 +1222,104 @@ QUnit.module('keyboard navigation', {
         delete this.animationStartAction;
     }
 }, () => {
+    QUnit.test('selectedIndex should be correct when one item is not visible using right arrow key', function(assert) {
```

Comment:
[testing coverage]
U've missed tests for `up/down` arrows.
I suggest u testing both "right" and "down" keys in the single test using forEach.
```javascript
['right', 'down'].forEach(key => {
    QUnit.test(`selectedIndex is updated correctly after pressing the ${key} arrow key if the second item is not visible`, function(assert) {
```

P.s. the same for "left" + "up"

Reply:
[84aed14](https://github.com/DevExpress/DevExtreme/pull/28137/commits/84aed14aa4791d6603352b53a1a6cc463c2f2d28)